server/common/Params.h

```diff
@@ -89,6 +90,12 @@ typedef int IncidentTrackerIdType;
 typedef uint64_t ActionLogIdType;
 #define FMT_ACTION_LOG_ID PRIu64
 
+typedef uint64_t LastInfoIdType;
```

Comment:
Why is this type is defined as 'uint64_t', although the corresponding column type in COLUMN_DEF_LAST_INFO is INT ?